src/pubsub/ua_pubsub_ns0.c

```diff
@@ -4,11 +4,17 @@
  *
  * Copyright (c) 2017-2018 Fraunhofer IOSB (Author: Andreas Ebner)
  * Copyright (c) 2019 Kalycito Infotech Private Limited
+ * Copyright (c) 2020 Yannick Wallerer, Siemens AG
+ * Copyright (c) 2020 Thomas Fischer, Siemens AG
  */
 
```

Comment:
I would prefer to include the term 'file' to the methods to add more clarity e.g. LoadPubSubConfigurationFile.